dhis-2/dhis-api/src/main/java/org/hisp/dhis/dataexchange/aggregate/TargetRequest.java

```diff
@@ -60,4 +62,10 @@ public class TargetRequest implements Serializable {
 
   /** Indicates whether to do a dry run. */
   @JsonProperty private Boolean dryRun;
+
+  /** Returns whether to skip audit, with fallback to default value if not set. */
+  @JsonIgnore
+  public boolean isSkipAuditOrDefault() {
+    return ObjectUtils.firstNonNull(skipAudit, true);
```

Comment:
Can't we make the `skipAudit` field be a `boolean` initialised to `true` ? That should have the same effect just will less code.

Reply:
I have bad experiences with primitive booleans in data models. Then you have no way of knowing whether the user supplied a value or not. I am also unsure how this works for existing Postgres JSONB blobs, i.e. if existing blobs have no `skipAudit` value, we may get a nullpointer here, not sure. Just wanted to keep it consistent with objects everywhere.

Reply:
I know what you're saying :)
For any new models, I think I'd advocate for either:
- primitive boolean with safe default (obviously this might not suit every use case) or
- use an enum instead of true/false (richer model)
I would always try to avoid having `Boolean`.